src/components/fields/MultipleChoice.js

```diff
@@ -145,41 +146,52 @@ class MultipleChoice extends AskField {
     return { options: selectedOptions }
   }
 
-  render () {
+  getCharIndex(i) {
+    return String.fromCharCode(65 + i);
```

Comment:
while I like what you're doing here because I'm a huge nerd, it might be more obvious to name this something like `getAlphaLabel` and just `return "ABCDEFGHIJKLMNOPQRSTUVWXYZ"[i]`